lib/rubocop/cop/rspec/factory_girl/dynamic_attribute_defined_statically.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryGirl
+        # Prefer to declare dynamic attribute values in a block.
+        #
+        # @example
+        #   # bad
+        #   kind [:active, :rejected].sample
+        #
+        #   # good
+        #   kind { [:active, :rejected].sample }
+        #
+        #   # bad
+        #   closed_at 1.day.from_now
+        #
+        #   # good
+        #   closed_at { 1.day.from_now }
+        #
+        #   # good
+        #   kind :static
+        #
+        #   # good
+        #   comments_count 0
+        #
+        #   # good
+        #   type User::MAGIC
+        class DynamicAttributeDefinedStatically < Cop
+          MSG = 'Use a block to set a dynamic value to an attribute.'.freeze
+
+          def_node_matcher :dynamic_defined_statically?, <<-PATTERN
+          (send nil _ (send ... ))
```

Comment:
I don't know if I get your point. The cop is looking only for calls that are not using a block. if it's using a block, it's already dynamic right?